rust/worker/benches/filter.rs

```diff
@@ -116,27 +114,17 @@ fn bench_filter(criterion: &mut Criterion) {
         generator: log_generator,
     };
 
-    let routine = |filter_input| async move {
-        FilterOperator::new()
-            .run(&filter_input)
-            .await
-            .expect("Filter should not fail.");
+    let routine = |_filter_input| async move {
+        // TODO: Run filter operator
     };
 
     for record_count in [1000, 10000, 100000] {
         let mut compact = CompactSegment::default();
         runtime.block_on(async { compact.populate_with_generator(record_count, &logen).await });
 
-        for (op, where_clause) in baseline_where_clauses() {
+        for (op, _where_clause) in baseline_where_clauses() {
             let setup = || {
-                FilterInput::new(
-                    compact.blockfile_provider.clone(),
-                    compact.record.clone(),
-                    compact.metadata.clone(),
-                    Chunk::new(Vec::new().into()),
-                    None,
-                    where_clause.clone(),
-                )
+                // TODO: Construct filter input
```

Comment:
are you planning to do this before merge? I didn't see this updated up stack

Reply:
I'm planning to add back the tests/bench before merge. That's the next thing I'll be work on after the CI passes for the PR that updates the grpc.

Reply:
Update: All bench and tests are restored for all operators involved in `GetOrchestrator`